.storybook/stories/modal/sidepanel.stories.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrSidepanel, ClrSidepanelModule, commonStringsDefault } from '@clr/angular';
+import { action } from '@storybook/addon-actions';
+import { moduleMetadata, StoryFn, StoryObj } from '@storybook/angular';
+
+import { CommonModules, removeFocusOutline } from '../../helpers/common';
+
+export default {
+  title: 'Modal/Side Panel',
+  decorators: [
+    moduleMetadata({
+      imports: [...CommonModules, ClrSidepanelModule],
+    }),
+  ],
+  component: ClrSidepanel,
+  argTypes: {
+    // inputs
+    clrSidepanelCloseButtonAriaLabel: { type: 'string', defaultValue: commonStringsDefault.close },
+    clrSidepanelLabelledById: { defaultValue: '' },
+    clrSidepanelSize: {
+      defaultValue: 'md',
+      options: ['sm', 'md', 'lg', 'xl', 'full-screen'],
+      control: { type: 'radio' },
+    },
```

Comment:
The `defaultValue` properties should be removed from `argTypes`. Instead, set the default value in `args`. This was part to the Storybook v8 update.